lib/MusicBrainz/Server/Form/Genre.pm

```diff
@@ -0,0 +1,19 @@
+package MusicBrainz::Server::Form::Genre;
+use HTML::FormHandler::Moose;
+
+extends 'MusicBrainz::Server::Form';
+
+has '+name' => ( default => 'edit-genre' );
+
+has_field 'name' => (
+    type => '+MusicBrainz::Server::Form::Field::Text',
+    required => 1,
+);
+
+has_field 'comment' => (
+    type => '+MusicBrainz::Server::Form::Field::Comment',
+);
+
+sub edit_field_names { qw( name comment ) }
+
+1;
```

Comment:
Can we add the license trailer here?

Reply:
Sure thing! We should add it to a lot of other places, but not forgetting it in the new files is probably a good start :D